CONTRIBUTING.md

```diff
@@ -22,3 +22,6 @@ Guidelines for pull requests:
 Before creating an icon request, please search to see if someone has requested the icon already. If there is an open request, please add a :+1:.
 
 If the icon has not already been requested, [create an issue](https://github.com/lucide-icons/lucide/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
+
+## Icon Requests from Feather
+If you are a designer who wants to contribute to Lucide but you don't know what icons to work on, then have a look at the Requests from Feather. All open, unfinished and valid requests can be found in [Fether Icon Requests](https://github.com/lucide-icons/lucide/issues/119).
```

Comment:
```suggestion
If you are a designer who wants to contribute to Lucide but you don't know what icons to work on, then have a look at the Requests from Feather. All open, unfinished and valid requests can be found in [Feaher Icon Requests](https://github.com/lucide-icons/lucide/issues/119).
```

Reply:
Maybe I'm blind but I don't see it, could you fill me in?

Reply:
Ok, got it now.

Reply:
You wrote `fether` but it is `feather`